apps/client/src/app/pages/home-page/FooterPage.tsx

```diff
@@ -0,0 +1,31 @@
+import Button from '@map-shared-components/Button';
+import SocialMediaButtons from '@src/pages/home-page/SocialMediaButton';
+
+function Footer({ oneClick }: FooterProps) {
+  return (
+    <footer className='bg-[rgb(29,60,102)] text-[#CADEED] py-1 px-3  mx-auto w-full]'>
+      <div className='container mx-auto flex justify-center  className=w-full sm:w-1/2 md:w-auto m-0.5'>
+        <div className='text-center font-roboto '>
+          <SocialMediaButtons />
+          <div>
+            <Button onClick={oneClick}>Info</Button>
```

Comment:
the same for these 3 buttons, maybe each could be named a support button or something else